### PR TITLE
sparkline V9

### PR DESCRIPTION
**Pull request by Anush2303**
## Previous Behavior

 Sparkline in V8 

## New Behavior

 Migrated Sparkline to v9  

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/react-components/react-charts-preview/package.json [hunk 1]
```diff
@@ -69,7 +69,8 @@
     "@types/react": ">=16.14.0 <19.0.0",
     "@types/react-dom": ">=16.9.0 <19.0.0",
     "react": ">=16.14.0 <19.0.0",
-    "react-dom": ">=16.14.0 <19.0.0"
+    "react-dom": ">=16.14.0 <19.0.0",
+    "@fluentui/react-components": "*"
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/package.json:73]: revert this line of change

File: packages/react-components/react-charts-preview/src/components/Sparkline/Sparkline.tsx [hunk 2]
```diff
@@ -1,16 +1,146 @@
...
+
+    const classes = useSparklineStyles_unstable(props);
+    const focusAttributes = useFocusableGroup();
+    return !_isChartEmpty() ? (
+      <div className={classes.inlineBlock} {...focusAttributes}>
+        {width >= 50 && height >= 16 ? (
+          <svg
+            width={width}
+            height={height}
+            data-is-focusable={true}
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/Sparkline/Sparkline.tsx:123]: remove data is focusable attribute. It is no longer needed

Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/Sparkline/Sparkline.tsx:123]: Okay

File: packages/react-components/react-charts-preview/package.json [hunk 3]
```diff
@@ -40,8 +40,8 @@
     "@fluentui/react-jsx-runtime": "^9.0.36",
     "@fluentui/react-shared-contexts": "^9.17.0",
     "@fluentui/react-theme": "^9.1.19",
-    "@fluentui/react-utilities": "^9.18.7",
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/package.json:43]: revert this change of removing react-utilities

File: packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx [hunk 4]
```diff
@@ -32,7 +32,7 @@ export const HorizontalBarChart: React.FunctionComponent<IHorizontalBarChartProp
   let _calloutAnchorPoint: IChartDataPoint | null;
   const _isRTL: boolean = isRtl();
   let barChartSvgRef: React.RefObject<SVGSVGElement> = React.createRef<SVGSVGElement>();
-  const _emptyChartId: string = useId('_HBC_empty');
+  const _emptyChartId: string = '_HBC_empty';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx:35]: revert this change

File: packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { ISparklineProps, ISparklineStyles } from './Sparkline.types';
+import type { SlotClassNames } from '../../../../react-utilities/src/index';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts:3]: Use @fluentui/react-utilities

File: packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import { tokens } from '@fluentui/react-theme';
+
+/**
+ * @internal
+ */
+export const hbcClassNames: SlotClassNames<ISparklineStyles> = {
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts:9]: call these sparklineClassNames - hbc stands for horizontal bar chart

File: packages/react-components/react-charts-preview/stories/Sparkline/SparklineDefault.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,393 @@
+import * as React from 'react';
+import { Sparkline, ISparklineProps } from '../../src/Sparkline';
+import { getColorFromToken, DataVizPalette } from '../../src/utilities/colors';
+
+export const HBCBasic = () => {
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/stories/Sparkline/SparklineDefault.stories.tsx:5]: Call this SparklineBasic

File: packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts (hunk 5 above)
Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts:3]: Where? for importing slotclassnames ?

File: packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts (hunk 6 above)
Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts:9]: Okay

**Comment by Anush2303** [comment]
Committed with all the changes.

File: packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Base Styles
+ */
+const useStyles = makeStyles({
+  inlineBlock: {
+    display: 'inline',
+  },
+  valueText: {
+    fontSize: tokens!.fontSizeBase200,
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts:22]: why is ! needed

Comment (Anush2303) [reply packages/react-components/react-charts-preview/src/components/Sparkline/useSparklineStyles.styles.ts:22]: Okay here there is no need. tokens would never be null. Will remove this.

**Review by AtishayMsft — APPROVED** [review]
